.github/workflows/restyled.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          ref: ${{ github.event.pull_request.head.sha }}
+          ref: ${{ github.event.pull_request.head.ref }}
```

Comment:
Did we mean to make this change? Restyled is failing on recent PRs....not clear to me why `git checkout` is failing.

Reply:
Seems related to this? https://github.com/getredash/redash/pull/7191#issuecomment-2414469929